.github/workflows/test.yml

```diff
@@ -42,5 +42,17 @@ jobs:
         run: npm run prettier:check
       - name: Test
         run: npm test
+      # N.B.: The selenium server cannot run as a GitHub Actions "service" as
+      # the tests need to mount local data into the container.
+      - name: Start Selenium Server
+        run: |
+          docker run -d -p 4444:4444 \
+            --shm-size="2g" \
+            --name selenium_server \
+            -v ${PWD}:${PWD} \
```

Comment:
So I am not sure if this is expected or not, but the driver only works if it has access to the local FS in order to read the `index.html` file. Is this ok for SauceLabs / BrowserStack et. al. scenarios?

Reply:
I was thinking, if we have a website hosted on GitHub pages, we could have the `index.html` file there. It could even be the default URL in case the host is not `localhost`.